### ISSUE TITLE
Why does VerticalAquaRadioButtonGroup specify a11yNameAttribute: 'verticalAquaButtonGroup' + instanceCount + 'Member'

### ISSUE THREAD

**Issue by samreid** [phetsims/sun#377]
Discovered while looking at https://github.com/phetsims/sun/issues/344

VerticalAquaRadioButtonGroup has:

```js
a11yNameAttribute: 'verticalAquaButtonGroup' + instanceCount + 'Member'
```

I can't figure out why it is important or useful for AquaRadioButton instances to have a name attribute like that.  Is it just important that they have unique names?  Why don't other components have something like this?  Is it a special requirement for radio button peers?

**Comment by jessegreenberg** [comment]
> Is it just important that they have unique names?

Thanks, that is correct. Each button in a group of radio buttons must have the same `name` attribute, or else the browser will treat all radio buttons in the document as in a single group and arrow keys will navigate through all of them. I added documentation about this in particular, @samreid does it look better?

**Comment by jessegreenberg** [comment]
Same needs to be done for RadioButtonGroup.js

**Comment by samreid** [comment]
Thanks, that is much clearer.  I'm seeing

AquaRadioButton:
```js
a11yNameAttribute: null 
```

VerticalAquaRadioButtonGroup:
```js
a11yNameAttribute: instanceCount
```

Will everything be OK if a null value gets set with `setAccessibleAttribute`, or do we need checking around that?  Also, this commit changed the value type from string to number, will that be OK?

**Comment by jessegreenberg** [comment]
non-string values are are automatically converted to string, so numbers are OK, but `null` is not OK for the same reason. Thanks, I added a check.

**Comment by samreid** [comment]
The check looks like it will work at the moment, but would fail if `instanceCount++;` was moved later in the `VerticalAquaRadioButtonGroup` constructor.   Perhaps it would be better to:

(a) default `a11yNameAttribute` to something other than null and remove the check
or
(b) check whether `a11yNameAttribute` is non-null instead of checking whether it is truthy

I suspect (b) may be better, but I'm not sure.  @jessegreenberg what do you think?

**Comment by jessegreenberg** [comment]
I see what you mean, I went with (b) that does seem best. Look OK @samreid?

**Comment by samreid** [comment]
Looks great, thanks, closing.
